app/utils/discogs.ts

```diff
@@ -1,25 +1,33 @@
 import { SearchResponseSchema } from "./discogs.schemas";
```

Comment:
I'd format this file, looks like prettier isn't auto-running for some reason :l

we might have to look at that together